### PR TITLE
Windows 10 bug fixes

### PR DESCRIPTION
**Pull request by moflo**
Hi, nice library! I ran into two issue using this on Anaconda / Python 3.7 on Windows 10, two minor fixes to `options.py` file. Thanks!

### CODE REVIEW

File: cottoncandy/options.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import os
 import sys
-import pwd
+# import pwd    # error on Windows 10, removed
```
Comment (anwarnunez) [inline cottoncandy/options.py]: I think we can delete this import all together. It's not used elsewhere.

File: cottoncandy/options.py [hunk 2]
```diff
@@ -71,7 +71,7 @@ def get_config():
     return config
 
 cwd = os.path.split(os.path.abspath(__file__))[0]
-userdir = appdirs.user_data_dir("cottoncandy")
+userdir = appdirs.user_data_dir("cottoncandy",appauthor="cottoncandy")  # error on Windows 10, appdirs requires appauthor
```
Comment (anwarnunez) [inline cottoncandy/options.py]: No need for the comment:
`userdir = appdirs.user_data_dir("cottoncandy",appauthor="cottoncandy") `

**Comment by moflo** [comment]
Sorry for the delay, thanks!

**Comment by anwarnunez** [comment]
I think you accidentally closed the pull request. 
Also, not sure I see new changes. Maybe something went wrong with the push? 
If you're busy, I don't mind merging and I can handle my minor comments at a later point.
Thanks!

**Pull request merged**